### PR TITLE
Prepare for 1.0.0-rc1 release

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Purpose
Prepare for 1.0.0-rc1 release

### Issues
Task

### Automation tests
 - Unit tests added: N/A
 - Integration tests added: N/A

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [-] Validated respective github issues
- [-] Assigned milestone to the github issue(s)

### CODE REVIEW

File: release-note.html [hunk 1]
```diff
@@ -28,17 +28,16 @@
 <body>
 <div id="main-content">
     <div>
-        <h1>What's new in 1.0.0-Beta</h1>
-        <p>This is the beta release of the Choreo Connect 1.0.0.</p>
+        <p>This is the RC1 release of the Choreo Connect 1.0.0.</p>
```
Comment (renuka-fernando) [inline release-note.html:31]: Should this be
```suggestion
        <p>Choreo Connect 1.0.0 is Released</p>
```
@praminda

File: release-note.html [hunk 2]
```diff
@@ -23,22 +23,21 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta content="text/html; charset=iso-8859-1" http-equiv="content-type"/>
-    <title>Choreo Connect v1.0.0-Beta - 22th November 2021</title>
+    <title>Choreo Connect v1.0.0-RC1 - 6th December 2021</title>
```
Comment (renuka-fernando) [inline release-note.html:26]:
```suggestion
    <title>Choreo Connect v1.0.0 - 6th December 2021</title>
```

Comment (renuka-fernando) [reply release-note.html:26]: @praminda this should be corrected right?

Comment (praminda) [reply release-note.html:26]: yes, anything in the src should say 1.0.0. no RC1 should be there

File: release-note.html (hunk 1 above)
Comment (praminda) [reply release-note.html]: +1

**Review by praminda — APPROVED** [review]

File: remove-containers.sh [hunk 3]
```diff
@@ -16,16 +16,16 @@
...
+docker kill "$(docker ps -a | grep wso2/choreo-connect-adapter:$cc_version | awk '{print $1}')"
+docker rm "$(docker ps -a | grep wso2/choreo-connect-adapter:$cc_version | awk '{print $1}')"
 
-docker kill "$(docker ps -a | grep wso2/choreo-connect-router:$micro_gw_version | awk '{print $1}')"
-docker rm "$(docker ps -a | grep wso2/choreo-connect-router:$micro_gw_version | awk '{print $1}')"
+docker kill "$(docker ps -a | grep wso2/choreo-connect-router:$cc_version | awk '{print $1}')"
+docker rm "$(docker ps -a | grep wso2/choreo-connect-router:$cc_version | awk '{print $1}')"
 
 docker kill "$(docker ps -a | grep wso2/choreo-connect-mock-backend:latest | awk '{print $1}')"
 docker rm "$(docker ps -a | grep wso2/choreo-connect-mock-backend:latest | awk '{print $1}')"
```
Comment (suksw) [inline remove-containers.sh:31]:
```suggestion
docker kill "$(docker ps -a | grep wso2/choreo-connect-mock-backend:$cc_version | awk '{print $1}')"
docker rm "$(docker ps -a | grep wso2/choreo-connect-mock-backend:$cc_version | awk '{print $1}')"
```

**Review by suksw — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211215.32

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211215.43

**Pull request merged**